load_tests/setup_test_environment.sh

```diff
@@ -14,6 +14,8 @@ stackOutputs=$(aws cloudformation describe-stacks --stack-name ${LOG_STORAGE_STA
 read -r -a outputArray <<< "$stackOutputs"
 export S3_BUCKET_NAME="${outputArray[0]}"
 
+export AWS_DEFAULT_REGION=${AWS_REGION}
```

Comment:
This probably isn't needed but it doesn't hurt. I added it sicne AWS_REGION is set in Code Build but it seems boto3 from local test liked AWS_DEFAULT_REGION